ui/commons/src/main/js/config/process/CommonConfig.js

```diff
@@ -12,14 +12,17 @@
  * information: "Portions copyright [year] [name of copyright owner]".
  *
  * Copyright 2011-2016 ForgeRock AS.
+ * Portions copyright 2025 3A Systems LLC.
  */
 
 define([
     "jquery",
     "lodash",
     "org/forgerock/commons/ui/common/util/Constants",
-    "org/forgerock/commons/ui/common/main/EventManager"
-], function($, _, Constants, EventManager) {
+    "org/forgerock/commons/ui/common/main/EventManager",
+    "org/forgerock/commons/ui/common/main/SpinnerManager",
+    "org/forgerock/commons/ui/common/main/ErrorsHandler"
+], function($, _, Constants, EventManager, spinner,  errorsHandler) {
```

Comment:
// double space in front of errorsHandler

Reply:
Fixed in 5895a66, thanks!